gc/verbose/schema.xsd

```diff
@@ -598,6 +599,13 @@ xmlns:vgc="http://www.ibm.com/j9/verbosegc">
 		<attribute name="bytes" type="integer" use="required" />
 	</complexType>
 
+	<complexType name="regions">
```

Comment:
Does the declaration order matter?

Reply:
no, it can be appended at the end of complexTypes